### DISCUSSION TITLE
pdf.js 4.2

### DISCUSSION THREAD

**Discussion by ignat-ukr** [stephanrauh/ngx-extended-pdf-viewer#2357] [Q&A] [answered]
Hello. I am waiting for a bug fix that is included into pdf.js version 4.2.67
When approximately it will reach the ngx-extended-pdf-viewer?
Thanks

**Comment by stephanrauh** [comment, marked as answer]
You'll simply have to update to version 20.5.0-alpha.*. I've called it an alpha version because I want to test it thoroughly, but at the moment, I still believe it's a high-quality version, so I encourage you to adopt it.
